jovian/utils/commit.py

```diff
@@ -168,6 +175,9 @@ def commit(message=None,
 
     log('Committed successfully! ' + urljoin(read_webapp_url(), username, title))
 
+    if in_jupyter_extension:
+        return urljoin(read_webapp_url(), username, title)
```

Comment:
```suggestion
    return urljoin(read_webapp_url(), username, title)
```

Reply:
we can always return this

Reply:
Jupyter prints the returned value implicitly. The check was to ensure that would not happen when you do `jovian.commit()` in notebook cell. I guess we need to that check back to avoid duplicate printing of the notebook url.